### ISSUE TITLE
Package details

### ISSUE THREAD

**Issue by sagikazarmark** [lstrojny/fxmlrpc#27]
Some package improvements I would do:

* Add a `.editorconfig` file for making compatible IDEs follow conventions.

Sample content:
```
root = true

[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[composer.json]
indent_style = space
indent_size = 4

[*.php]
indent_style = space
indent_size = 4

[*.yml]
indent_style = space
indent_size = 2
```

* Add a `.scrutinizer.yml` file, to customize tests and ignore `tests/` directory.

Sample content:
``` yml
filter:
  paths: [src/*]
tools:
  php_analyzer: true
  php_mess_detector: true
  php_pdepend: true
  external_code_coverage:
    timeout: '600'
```

* Add code coverage upload in travis using ocular.

Sample content:
``` yml
language: php

php:
  - 5.4
  - 5.5
  - 5.6
  - hhvm

before_script:
  - travis_retry composer self-update
  - travis_retry composer install --prefer-source --no-interaction
  - npm install

script: phpunit

after_script:
  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then wget https://scrutinizer-ci.com/ocular.phar; fi
  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml; fi
```

(Of course it needs a phpunit configuration generating coverage)

If you like any of these, I am happy to open a PR.

**Comment by lstrojny** [comment]
Sure, go ahead, looking forward to it!

**Comment by sagikazarmark** [comment]
Closed by #28
